en/datastore/guidance/data-formats.rst

```diff
@@ -10,12 +10,12 @@ When outputting data in CSV format you can instead choose to display data based
 XML
 ---
 
-The Datastore returns the original activity XML as published. This is :doc:`enhanced with metadata </datastore/reference/structure-of-data>` specifying the version that individual activities were published at, as well as details of the query result.
+The Datastore returns the original activity XML as published. This is `enhanced with metadata <http://iatistandard.org/202/guidance/datastore/reference/structure-of-data/>`__ specifying the version that individual activities were published at, as well as details of the query result.
```

Comment:
Links shouldn't include the version number - when it's missing it auto-directs to the latest.

Reply:
woops